Doc/data/refcounts.dat

```diff
@@ -1849,6 +1849,9 @@ PyObject_RichCompareBool:PyObject*:o1:0:
 PyObject_RichCompareBool:PyObject*:o2:0:
 PyObject_RichCompareBool:int:opid::
 
+PyObject_SelfIter:PyObject*::+1:
+PyObject_SelfIter:PyObject*:obj:0:
```

Comment:
I want to have the text *Return value: New reference.* show up in the documentation, but I failed. Perhaps this is wrong?

Reply:
OK, it works on https://cpython-previews--127861.org.readthedocs.build/en/127861/c-api/object.html#c.PyObject_SelfIter -- perhaps a cache thing locally.